stories/guides/circlieci-cli.stories.mdx

```diff
@@ -44,7 +44,14 @@ Once you have installed the CircleCI CLI, you can simply open your VScode, and p
 `Ctrl`/`Cmd` +`Shift`+`P`
 and type; `‘Run task’`. 
 
-Hit enter and you will then see the `CircleCI CLI task` pop up. 
-Just hit enter again and then once more on the option, `"Continue without scanning the task output"` to run the CircleCI build. 
+Hit enter and you will then see a list of `CircleCI` tasks.
 
-This can take around 15-20 mins to complete the run.
\ No newline at end of file
+Currently we have (and what they run);
+`CircleCI Complete Build` - Full CircleCI workflow
+`CircleCI Build` - Build steps of the CircleCI workflow
+`CircleCI Lint` - Linting steps of the CircleCI workflow
+`CircleCI Unit` - Build and unit testing steps of the CircleCI workflow
```

Comment:
Update the build name here to be `CircleCI Unit + Integration` - The description also needs to be updated, as this doesn't do a build.